frontend/src/components/Game.js

```diff
@@ -88,6 +93,8 @@ export default function Game({gameId, clientUsername, initialPlayer, playerNames
                                   diceResult={diceResult} selectedWord={selectedWord}
                                   currentPlayer={currentPlayer} clientUsername={clientUsername}
                                   sendWebSocketMessage={sendMessage}/>
+    }else if(gamePhase == GamePhase.GAME_OVER){
```

Comment:
```suggestion
    } else if (gamePhase === GamePhase.GAME_OVER) {
```

Note the triple equals `===`. Intellij will try to warn you about this by highlighting it in yellow.